src/Components/Registration/Form/index.tsx

```diff
@@ -44,7 +45,7 @@ const RegistrationFormImpl: React.FC<IRegistrationForm.IProps> = ({ t }): JSX.El
         label={t('Registration_Username')}
         name="username"
         validateFirst
-        rules={Rules.username}
+        rules={Rules(t).username}
```

Comment:
Please create a `rules` variable at the top instead of calling the function for multiple times.